core/src/main/resources/hudson/Messages_fr.properties

```
@@ -49,16 +47,82 @@ Util.day   ={0} j
 Util.year  ={0} an.
 
-UpdateCenter.DownloadButNotActivated=La mise à jour a été téléchargée avec succès. Elle sera activée lors du prochain démarrage
-
-FilePath.TildaDoesntWork=''~'' n''est supporté que sur les shells Unix.
+FilePath.TildaDoesntWork=’~’ n''est supporté que sur les shells Unix.
 
+PluginManager.DisplayName=Plugins
 PluginManager.PortNotANumber=Le port n''est pas un nombre
 PluginManager.PortNotInRange=Le port n''est pas dans l''intervalle de {0} à {1}
-AboutJenkins.DisplayName=A propos de Jenkins
+PluginManager.UploadPluginsPermission.Description=\
+  Obsolète - Veuillez utiliser le droit Overall/Administer à la place
+PluginManager.ConfigureUpdateCenterPermission.Description=\
+  Obsolète - Veuillez utiliser le droit Overall/Administer à la place
+PluginManager.PluginCycleDependenciesMonitor.DisplayName=Détecteur de dépendances cycliques
+PluginManager.PluginUpdateMonitor.DisplayName=Configuration de plugin invalide
+PluginManager.PluginDeprecationMonitor.DisplayName=Moniteur de plugin obsolète
+PluginManager.CheckUpdateServerError=Aucune erreur lors de la vérification du site de mise à jour : {0}
+PluginManager.UpdateSiteError=Erreur lors de la vérification des sites de mise à jour pour {0} essai(s). La dernière exception était : {1}
+PluginManager.UpdateSiteChangeLogLevel=Modifiez le niveau de log du logger {0} à WARNING ou moins pour consulter plus d''informations et le message d''erreur de chaque essai
+PluginManager.UnexpectedException=Exception inattendue lors la vérification des serveurs de mise à jour
+
+
+PluginManager.compatWarning=\
+ Avertissement : la nouvelle version de ce plugin est indiquée comme incompatible avec la version installée. \
+ C''est habituellement le cas parce que son comportement ou ses APIs ont changé, ou parce qu''il utilise un format de configuration différent de celui installé. \
+ D''autres plugins ayant une dépendance à ce plugin sont peut-être incompatibles avec cette mise à jour et ne fonctionneront plus comme attendu, les jobs utilisant ce plugin necessiteront peut-être d''être reconfigurés, et/ou vous ne serez peut-être plus capable de revenir à la version précédente sans avoir à restaurer manuellement les configurations précédentes. \
+ Veuillez consulter la note de livraison du plugin pour plus de détails.
+PluginManager.parentDepCompatWarning=Les plugins suivants sont incompatibles :
+PluginManager.parentCompatWarning=Les plugins suivants sont affectés par cela :
+PluginManager.coreWarning=\
+ Avertissement : ce plugin est compatible pour Jenkins {0} ou plus récent. \
+ Jenkins ne démarrera pas ce plugin s''il est installé.
+PluginManager.depCompatWarning=\
+ Avertissement : ce plugin nécessite des versions plus récentes de dépendances et au moins un de ces plugins n''est pas compatible avec la version installée. \
+ C''est habituellement le cas parce que son comportement ou ses APIs ont changé, ou parce qu''il utilise un format de configuration différent de celui installé. \
```

```suggestion
 C''est habituellement le cas parce que le comportement de ce plugin de dépendance ou ses APIs ont changé, ou parce qu''il utilise un format de configuration différent de celui installé. \
```
To make the link (cf suggestion above)